resilience4j-spring-boot2/src/main/java/io/github/resilience4j/bulkhead/autoconfigure/BulkheadMetricsAutoConfiguration.java

```diff
@@ -34,7 +36,7 @@
  */
 @Configuration
 @ConditionalOnClass({MeterRegistry.class, Bulkhead.class, TaggedBulkheadMetricsPublisher.class})
-@AutoConfigureAfter(MetricsAutoConfiguration.class)
+@AutoConfigureAfter({MetricsAutoConfiguration.class, SimpleMetricsExportAutoConfiguration.class})
```

Comment:
I think it works well only with `MetricsAutoConfiguration.class`.

Reply:
Hi @dlsrb6342 , As per your previous comment I understood we need to add ```SimpleMetricsExportAutoConfiguration.class``` to ```@AutoConfigureAfter``` in **MetricsAutoConfiguration. please refer to the previous comment below:
https://github.com/resilience4j/resilience4j/pull/725#discussion_r353803397
Kindly let me know if I have misunderstood something.

Reply:
Oh Sorry. Please ignore this comment.